samples/node/slack/README.adoc

```diff
@@ -0,0 +1,148 @@
+= slack-command
+
+[horizontal]
+*Language*:: JavaScript
+*Function*:: https://github.com/projectriff/riff/blob/master/samples/slack/slack-command/slack_command.js[slack_command.js]
+*Protocol*:: HTTP
+*Input*:: slack
+*Output*:: none (replies, implicitly)
+
+== Deploy the FaaS
+
+[NOTE]
+This example requires a Kubernetes cluster with LoadBalancer support and a public facing IP address such as https://cloud.google.com/kubernetes-engine/[GKE].
+Alternately, you can use an http tunnel such as https://ngrok.com/[ngrok].
+
+Deploy riff with link:../../../Getting-Started.adoc[helm].
+
+== Running on minikube with ngrok
+
+Once riff is running and ngrok is installed, get the URL of the http-gateway:
+
+```
+export NODE_PORT=$(kubectl get --namespace default -o jsonpath="{.spec.ports[0].nodePort}" services demo-riff-http-gateway)
+export NODE_IP=$(kubectl get nodes --namespace default -o jsonpath="{.items[0].status.addresses[0].address}")
+echo http://$NODE_IP:$NODE_PORT
+```
+You should see something like:
+```
+http://192.168.99.100:32595
+```
+
+In a separate terminal, start ngrok (substituting your actual URL):
+```
+ngrok http 192.168.99.100:32595
+```
+
+You should see some output including something like:
+
+```
+Forwarding                    http://a61d8bb0.ngrok.io -> 192.168.99.100:32595
+```
+
+=== Build the Docker container for the function code
+
+```
+eval $(minikube docker-env)
+docker build -t projectriff/slack-command:test samples/node/slack/slack-command
+```
+=== Create the Function and Topic(s)
+
+```
+./riff apply -f samples/node/slack/slack-command
+```
+
+=== Test the function locally
+
+Since the input topic configured for the function is named `slack`, configure the slash command to `POST` to `http://<ngrokid>.ngrok.io/requests/slack`.
```

Comment:
Without recreating Slash's own documentation, we should at least explain that the user needs to create a Slack command (+ link to where to start on their site / doco).
This sentence states "the slash command" but I as a reader have no idea what it is (unless I missed something)